fossology/fossology
# chore(cmake): do not cache git version

## Description

1. Do not cache git version information in cmake, allowing rebuild of VERSION files at each commit change using `cmake --build` command.
2. Clean TWIG cache while installing `www` packages preventing confusion while working on UI.

### Changes

1. Remove git variables from cmake cache.
2. Add new install target in `www` to clean twig cache.

## How to test

1. Change git commit and run `cmake --build`, the VERSION files do not update showing old version information in UI.
2. With the commit, every change in git commit should cause rebuild of VERSION files.
3. At each install, the twig cache should get cleaned (`/var/local/cache/fossology`)

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2362"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
